examples/mlflow_artifacts/README.md

```diff
@@ -0,0 +1,45 @@
+This directory contains a set of files for demonstrating the MLflow Artifacts service.
```

Comment:
Can we explain what the artifacts service does and provide a small example command that users could run manually to serve artifacts via `mlflow server` in the README?

Reply:
Updated the README!